cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/DcLineSegmentConversion.java

```diff
@@ -11,16 +11,20 @@
 
 import com.powsybl.cgmes.conversion.Context;
 import com.powsybl.cgmes.model.CgmesNames;
+import com.powsybl.commons.PowsyblException;
 import com.powsybl.iidm.network.HvdcConverterStation;
 import com.powsybl.iidm.network.HvdcLine;
 import com.powsybl.iidm.network.HvdcLineAdder;
+import com.powsybl.iidm.network.LccConverterStation;
 import com.powsybl.triplestore.api.PropertyBag;
 
 /**
  * @author Luma Zamarreño <zamarrenolm at aia.es>
  */
 public class DcLineSegmentConversion extends AbstractIdentifiedObjectConversion {
 
+    private static final String TARGET_PPCC = "targetPpcc";
```

Comment:
Leave a comment to define "targetPpcc":
// Real power injection target in the AC grid.